shared/src/commonMain/kotlin/com/example/tasky/agenda/domain/model/AgendaItem.kt

```diff
@@ -118,6 +118,72 @@ data class Event(
     override fun getStartTime(): Long = from
 }
 
+@Serializable
+data class Attendee(
```

Comment:
Why do you need your domain model to be serializable here? 🤔
Usually, domain models are plain and simple - UI models and data models are usually parcelable or serializable.